src/locale/locales/pt-BR/messages.po

```diff
@@ -577,33 +577,33 @@ msgstr "Um email foi enviado para seu email anterior, {0}. Ele inclui um código
 
 #: src/components/dialogs/GifSelect.tsx:254
 msgid "An error has occurred"
-msgstr ""
+msgstr "Ocorreu um erro"
 
 #: src/components/dialogs/GifSelect.tsx:252
 #~ msgid "An error occured"
 #~ msgstr "Tivemos um problema"
 
 #: src/view/com/util/post-embeds/VideoEmbedInner/VideoWebControls.tsx:314
 msgid "An error occurred"
-msgstr ""
+msgstr "Ocorreu um erro"
 
 #: src/components/StarterPack/ProfileStarterPacks.tsx:315
 msgid "An error occurred while generating your starter pack. Want to try again?"
-msgstr ""
+msgstr "Ocorreu um erro ao gerar seu pacote inicial. Quer tentar novamente?"
```

Comment:
I noticed that in other places "starter pack" was already translated as "kit inicial". We should probably follow the same convention in other translations

Reply:
ok